modules/multi-arch-creating-podplacmement-config-using-cli.adoc

```diff
@@ -0,0 +1,86 @@
+//Module included in the following assemblies
+//
+//post_installation_configuration/multiarch-tuning-operator.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="multi-architecture-creating-podplacement-config-using-cli_{context}"]
+
+= Creating a PodPlacementConfig CR by using the CLI
+
+You can create a `PodPlacementConfig` custom resource (CR) by using the {oc-first}.
+
+.Prerequisites
+
+* You have access to the cluster with `cluster-admin` privileges.
+* You have installed the {oc-first}.
+* You have installed the Multiarch Tuning Operator.
+
+.Procedure
+
+. Log in to the {oc-first}.
+
+. Create a `PodPlacementConfig` CR YAML file:
++
+.Example `PodPlacementConfig` CR YAML file with opt-in configuration
+[source,yaml]
+----
+apiVersion: multiarch.openshift.io/v1alpha1
+kind: ClusterPodPlacementConfig
+metadata:
+  name: cluster
+spec:
+  namespaceSelector:
+    matchExpressions:
+      - key: multiarch.openshift.io/include-pod-placement <1>
+        operator: Exists <2>
+----
++
+.Example `PodPlacementConfig` CR YAML file with opt-out configuration
+[source,yaml]
+----
+apiVersion: multiarch.openshift.io/v1alpha1
+kind: ClusterPodPlacementConfig
+metadata:
+  name: cluster
+spec:
+  logVerbosityLevel: Normal
+  namespaceSelector:
+    matchExpressions:
+      - key: multiarch.openshift.io/exclude-pod-placement <1>
+        operator: DoesNotExist <2>
+----
+<1> Set this field to either `multiarch.openshift.io/include-pod-placement` (opt-in configuration) or `multiarch.openshift.io/exclude-pod-placement` (opt-out configuration).
```

Comment:
I think it creates some confusion to split the opt-in vs opt-out configuration along multiple points. Maybe we can add a note on the `namespaceSelector` or `matchExpression` field and discuss of the two possible matchExpression to run opt-in or opt-out together.

Or, even better, we can have a note before the two examples to describe what we mean by opt-in or opt-out (I would also find another name for them).